tests/e2e/test_live_server.py

```
@@ -397,6 +397,61 @@ def test_platform_info(c: httpx.Client, h: dict) -> None:
+    r = req("POST", c, "/api/company", headers=h,
+            json={"name": "E2E Co", "domain": domain})
+    check(r.status_code == 201, "POST /api/company (valid body) → 201", r)
+    body = r.json()
+    company = body.get("company", body) if isinstance(body, dict) else body
+    cid = company.get("id")
+    check(bool(cid), "create response must include company id", r)
+    ok(f"POST /api/company → 201 (id={cid})")
+
+    # Read back — exercises the Mongo extra-field read path (create-500 guard).
+    r = req("GET", c, f"/api/company/{cid}", headers=h)
+    check(r.status_code == 200, f"GET /api/company/{{id}} → 200", r)
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)